dsaps/models.py

```diff
@@ -186,19 +189,21 @@ class Item(BaseRecord):
     file_identifier = Field()
     source_system_identifier = Field()
 
-    def bitstreams_from_directory(self, directory, file_type='*'):
+    def get_bitstreams_in_directory(self, directory, file_type='*'):
+        """Get bitstreams from the specified directory and sort them."""
         files = glob.iglob(
             f'{directory}/**/{self.file_identifier}*.{file_type}',
             recursive=True
-            )
+        )
         self.bitstreams = [
             Bitstream(name=os.path.basename(f),
                       file_path=f) for f in files
-            ]
+        ]
         self.bitstreams.sort(key=lambda x: x.name)
 
     @classmethod
-    def from_row(cls, row, field_map):
+    def create_item_metadata_from_csv_row(cls, row, field_map):
```

Comment:
Since this is a method on `Item`, I don't think the name needs to specify that it's item metadata. `Item.metadata_from_csv_row_` is just as clear and less redundant.

Reply:
Great point